charger/ocpp/cp.go

```diff
@@ -128,3 +131,97 @@ func (cp *CP) Connected() bool {
 func (cp *CP) HasConnected() <-chan struct{} {
 	return cp.connectC
 }
+
+// cp actions
+
+func (cp *CP) TriggerResetRequest(resetType core.ResetType) error {
```

Comment:
das brauchts gar nicht mehr

Reply:
Hat es dann wohl noch nie. War halt da und ich hab es einfach erstmal stumpf umgezogen.

Reply:
Ist auf jeden Fall eine Altlast. In 2022 gabs das Ding schon...